extensions/vertx-http/deployment/src/main/resources/dev-templates/io.quarkus.quarkus-vertx-http/test.html

```diff
@@ -85,7 +85,7 @@
         </div>
     </div>
     {/for}
-    {#for cn in info:tests.results.skipped}
+    {#for cn in info:tests.results.skipped.orEmpty}
```

Comment:
What does `orEmpty` do? Can't Qute deal with empty collections?

Reply:
If you open the page before the tests has run you get a nice error. This is how qute suggest fixing it.